### DISCUSSION TITLE
Can we have just "next", not "next_line_unless_newline_at_signature_end"?

### DISCUSSION THREAD

**Discussion by cbrichau** [PHP-CS-Fixer/PHP-CS-Fixer#6846] [Q&A] [answered]
## Feature request

Many braces allow a `next_line_unless_newline_at_signature_end` rule, but not simply `next`.

This conflicts with other tools that are all or nothing (i.e. always same line, or always next), such as beautifiers on save, or PHP-CS.

Would it be possible to have a `next` option for braces? E.g.:
- control_structures_opening_brace
- functions_opening_brace
- anonymous_functions_opening_brace
- classes_opening_brace
- anonymous_classes_opening_brace

**Comment by Wirone** [comment, marked as answer]
Yes, we can have such an option, the only thing needed is someone to implement it 😉. Are you interested in rising PR?

**Comment by Renji-FR** [comment]
Is it related to this fix?

### Example for `classes_opening_brace`

With the `next_line_unless_newline_at_signature_end` option:
```php
class MyClass
{
      public function test(
            string $arg1,
            string $arg2,
      }: bool {
            return false;
      }
}
```

The goal is to have this format/design with the `next`/`next_line` option:
```php
class MyClass
{
      public function test(
            string $arg1,
            string $arg2,
      }: bool
      {
            return false;
      }
}
```

**Comment by adrolter** [comment]
Is there an existing PR for this yet?
